android/res/values-fr/strings.xml

```diff
@@ -37,6 +37,17 @@
 	<string name="country_status_download_failed">Échec lors du téléchargement</string>
 	<!-- Button text for the button under the country_status_download_failed message -->
 	<string name="try_again">Réessayer</string>
+	<string name="about_menu_title">À propos de Organic Maps</string>
+	<!-- Text in About screen -->
+	<string name="about_headline">Gratuit pour tout le monde, fait avec amour</string>
+	<!-- Text in About screen -->
+	<string name="about_proposition_1">• Pas d\'annonces, pas de suivi, pas de collecte de données</string>
+	<!-- Text in About screen -->
+	<string name="about_proposition_2">• Pas de dédnuage de la batterie, pas de données de fond</string>
```

Comment:
Suggestion : "Ne draine pas la batterie, pas de consommation de données en arrière plan"

Reply:
My bad, here is a shorten translation : "Ne draine pas la batterie, pas besoin de données mobile"

Reply:
I am afraid that many other translations won't be short enough. The length should be less important than the _message_ we want to deliver.